R/inc_prop.R

```diff
@@ -0,0 +1,35 @@
+#' Computing the Incidence Proportion
+#'
+#' @param data (`data.frame`)\cr with columns including
```

Comment:
You can now use the new `@typed` syntax, see the existing function example in the package